sidebars.json

```diff
@@ -265,6 +263,15 @@
         }
       ]
     },
+    {
+      "type": "category",
+      "label": "Observability",
```

Comment:
For reviewers: an overview for Observability is not in the scope of this PR. It will be added later.